noetic/distribution.yaml

```diff
@@ -11079,6 +11079,21 @@ repositories:
       url: https://github.com/wjwwood/serial.git
       version: main
     status: maintained
+  serl_franka_controllers:
+    doc:
+      type: git
+      url: https://github.com//rail-berkeley/serl_franka_controllers.git
+      version: main
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com//rail-berkeley/serl_franka_controllers.git
+      version: 0.1.0-1
```

Comment:
This doesn't look right.  Was this generated by bloom or did you do it manually?

I would expect your release repo to be different from your source repo, and contain the necessary bloom-generated artifacts for the buildfarm to be able to generate binary packages.